.cpyskip.txt

```diff
@@ -107,6 +107,11 @@ docs/ide4c/vscode/*.json
 tools/trac/IcuCodeTools/*
 tools/unicodetools/*
 #
+# CLDR - TODO(review): verify that _readme.txt copyright notice is sufficient?
+#
+testdata/cldr-testData/transforms/*.txt
+testdata/cldr-testData/segmentation/graphemeCluster/*.txt
+#
```

Comment:
Since we're now using an include-list, these txt files aren't copied anymore. I'll remove this file from the PR before squashing into a single commit.